docker-compose.yaml

```diff
@@ -40,5 +40,7 @@ services:
       POSTGRES_DB: timelessdb_test
       POSTGRES_USER: timeless_user
       POSTGRES_PASSWORD: timeless_pwd
+  sync_worker:
+    command: bash -c 'celery -A timeless.sync.celery worker'
```

Comment:
@paulodamaso  I think you should have just left a puzzle to fix docker setup. But generally It should be:
```
  sync_worker:
    build: .
    command: bash -c 'celery -A timeless.sync.celery worker'
    environment:
      FLASK_APP: main.py
      FLASK_ENV: development
      SQLALCHEMY_DATABASE_URI: postgresql://timeless_user:timeless_pwd@db/timelessdb_dev
    volumes:
      - .:/usr/app
```

Reply:
@paulodamaso I just realized that it's a bit incorrect anyway. `-A timeless.sync.celery` means that in `timeless/sync/__init__.py` has to contain `celery` instance. I think you need to leave a puzzle to continue setup `celery`. Since it's still not ready to use.